daisy_workflows/image_import/ubuntu/translate.py

```diff
@@ -107,9 +107,9 @@ def DistroSpecific(g):
 
   # Try to reset the network to DHCP.
   if ubu_release == 'trusty':
-    g.write('/etc/network/intefaces', trusty_network)
```

Comment:
#341 introduced this, and I don't think it would work...

Reply:
You're right, please change it to /etc/network/interfaces